src/io/seek_from.rs

```diff
@@ -27,4 +27,22 @@ pub enum SeekFrom {
     /// to seek before byte 0.
     #[cfg_attr(staged_api, stable(feature = "rust1", since = "1.0.0"))]
     Current(#[cfg_attr(staged_api, stable(feature = "rust1", since = "1.0.0"))] i64),
+
+    /// Sets the offset to next location in the file greater than or equal to the
+    /// specified number of bytes that contains data.
+    ///
+    /// If the offset points to data, the file offset is set to it.
+    #[cfg_attr(staged_api, stable(feature = "rust1", since = "1.0.0"))]
+    Data(#[cfg_attr(staged_api, stable(feature = "rust1", since = "1.0.0"))] i64),
```

Comment:
The `cfg_attr` things in the other arms are there because this is code derived from std, which has these stability feature attributes. For our own arms, `Data` and `Hole`, we can leave of these `#[cfg_attr(...*)]` attributes entirely.